bindings/python/test.py

```diff
@@ -57,6 +57,53 @@ def test_torrent_handle(self):
 		self.h.prioritize_pieces([(0, 1)])
 		self.assertEqual(self.h.piece_priorities(), [1])
 
+	def test_unknown_torrent_parameter(self):
+		self.ses = lt.session({'alert_mask': lt.alert.category_t.all_categories,
+			'enable_dht': False})
+		try:
+			self.h = self.ses.add_torrent({'storage': ''})
```

Comment:
it's not obvious to me why this should be inaccessible. sure, it can't actually be set to a function pointer because of the problems of calling back into python code from libtorrent, but it would be possible for it to be set to some other object to select between the ``default_storage``, ``zero_storage`` and ``discard_storage``. similar to how plugins can be selected from python